api/util/auditLog.js

```diff
@@ -1,16 +1,17 @@
 const AuditLog = require('../main_endpoints/models/AuditLog');
+const logger = require('../util/logger');
 
-function logAudit({ userId, action, documentId = null, details = {} }) {
+async function logAudit({ userId, action, documentId = null, details = {} }) {
```

Comment:
we would be doing

```js
await logAudit({
      userId,
      action,
      documentId,
      details,
    });
```

when this is basically the same amount of lines as
```js
await AuditLog.create({
      userId,
      action,
      documentId,
      details,
    });
```

should we not define a helper function and instead call `AuditLog` directly?